test_scripts/RC/InteriorVehicleData_cache/033_GetInteriorVD_subscribe_false_by_app_unregistration_2_modules.lua

```diff
@@ -0,0 +1,82 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0178-GetInteriorVehicleData.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case
+-- 1. Mobile app1 is subscribed to module_1
+-- 2. Mobile app1 is subscribed to module_2
+-- 3. Mobile app1 unregisters
+-- SDL must
+-- 1. send GetInteriorVD(module_1, subscribe = false) request to HMI
+-- 2. send GetInteriorVD(module_2, subscribe = false) request to HMI
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/RC/InteriorVehicleData_cache/common_interiorVDcache')
+local commonRC = require('test_scripts/RC/commonRC')
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+
+-- [[ Local Function ]]
+local function unregistrationApp()
+  local rpc = "GetInteriorVehicleData"
+  EXPECT_HMICALL(commonRC.getHMIEventName(rpc))
+  :Do(function(_, data)
+      if data.params.moduleType == "CLIMATE" then
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS",
+        common.getHMIResponseParams(rpc, "CLIMATE", false))
+      else
+        common.getHMIConnection():SendResponse(data.id, data.method, "SUCCESS",
+        common.getHMIResponseParams(rpc, "RADIO", false))
+      end
+    end)
+  :ValidIf(function(_, data)
+      local isError = false
+      local ExpectedResult
+      if data.params.moduleType == "CLIMATE" then
+        ExpectedResult = common.getHMIRequestParams(rpc, "CLIMATE", 1, false)
+      else
+        ExpectedResult = common.getHMIRequestParams(rpc, "RADIO", 1, false)
+      end
+      isError = commonFunctions:is_table_equal(data.params, ExpectedResult)
+      if isError == false then
```

Comment:
@GetmanetsIrina `isError` is redundant and can be removed. Please use `commonFunctions:is_table_equal(data.params, ExpectedResult)` directly as condition.